tee-worker/litentry/core/credentials/src/credential_schema.rs

```diff
@@ -112,7 +112,7 @@ pub fn get_schema_url(assertion: &Assertion) -> Option<String> {
 		Assertion::NftHolder(_) => Some(format!("{BASE_URL}/26-nft-holder/1-1-1.json")),
 
 		Assertion::TokenHoldingAmount(_) =>
-			Some(format!("{BASE_URL}/25-token-holding-amount/1-1-2.json")),
+			Some(format!("{BASE_URL}/25-token-holding-amount/1-1-3.json")),
```

Comment:
Is it updated already?

Reply:
yes, otherwise the single-worker test will fail.